src/jdk.attach/windows/native/libattach/VirtualMachineImpl.c

```diff
@@ -231,24 +231,17 @@ JNIEXPORT jlong JNICALL Java_sun_tools_attach_VirtualMachineImpl_openProcess
     }
 
     /*
-     * On Windows 64-bit we need to handle 32-bit tools trying to attach to 64-bit
-     * processes (and visa versa). X-architecture attaching is currently not supported
-     * by this implementation.
+     * On Windows we need to handle 32-bit tools trying to attach to 64-bit
+     * processes, which is currently not supported by this implementation.
      */
```

Comment:
The tool side uses the attach API so the potential scenario is a tool on 64-bit attempting to attach to a target VM that is 32-bit. So the comment needs to re-phased to the reverse of what it says now.